### PR TITLE
Make dracula work with diffchar

### PR DESCRIPTION
**Pull request by ferdinandyb**
Screenshots and background in this issue:
https://github.com/rickhowe/diffchar.vim/issues/25

Code is by @rickhowe, not me.

Thanks for considering!

### CODE REVIEW

File: colors/dracula.vim [hunk 1]
```diff
@@ -207,15 +207,16 @@ call s:h('StatusLineTermNC', s:none, s:bglight)
...
+call s:h('DiffDelete', s:red, s:bgdark)
+
 hi! link ColorColumn  DraculaBgDark
 hi! link CursorColumn CursorLine
 hi! link CursorLineNr DraculaYellow
-hi! link DiffAdd      DraculaGreen
 hi! link DiffAdded    DiffAdd
-hi! link DiffChange   DraculaDiffChange
-hi! link DiffDelete   DraculaDiffDelete
 hi! link DiffRemoved  DiffDelete
```
Comment (dsifford) [inline colors/dracula.vim:219]: What about `DiffAdded` and `DiffRemoved`? Do those not need to be highlighted this way?

**Review by dsifford — COMMENTED** [review]
Thanks for the PR. Just a quick question, but otherwise LGTM

File: colors/dracula.vim (hunk 1 above)
Comment (ferdinandyb) [reply colors/dracula.vim:219]: I don't really know the internals, so all I can say, that in practice they do not :) For the sake of symmetry they could be, but I don't really know if there are any downsides you'd need to consider.

Comment (dsifford) [reply colors/dracula.vim:219]: Hmm yeah, can you just make those the same as the other diff ones just for future-proofing?

Comment (ferdinandyb) [reply colors/dracula.vim]: should be done

**Review by dsifford — APPROVED** [review]
LGTM. 

Any last min thoughts, @benknoble?

**Comment by benknoble** [comment]
I think this is really a temporary workaround, right? The real issue is that the other plugin didn't handle linked highlight groups correctly.

IIRC from threads on the mailing list, some folks are working on improving the vimscript API for handling highlights. Perhaps the plugin author needs to check that work and fix the plugin?

OTOH, I'm not really opposed to un-linking. Just want to make sure we're not taking on extra work for an out-of-band or out-of-scope issue.

**Comment by benknoble** [comment]
Is this still something that we need? Did the upstream issue get resolved, and either way is this something we should consider merging?

**Comment by ferdinandyb** [comment]
sorry for the late answer, tbh, I'm not sure what the situation is, I'll try to get around to checking this this week

**Comment by ferdinandyb** [comment]
The diffchar plugin sorted this on their side, everything works as expected from dracula and diffchar master branches so this PR is now obsolete. Thanks!

**Pull request closed without merging**